pkg/clusteragent/clusterchecks/status.go

```diff
@@ -0,0 +1,67 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package clusterchecks
+
+import (
+	"embed"
+	"io"
+
+	"github.com/DataDog/datadog-agent/comp/core/status"
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// Provider provides the functionality to populate the status output
+type Provider struct{}
+
+//go:embed status_templates
+var templatesFS embed.FS
+
+// Name returns the name
+func (Provider) Name() string {
+	return "Cluster Checks Dispatching"
+}
+
+// Section return the section
+func (Provider) Section() string {
+	return "Cluster Checks Dispatching"
+}
+
+// JSON populates the status map
+func (Provider) JSON(_ bool, stats map[string]interface{}) error {
+	populateStatus(stats)
+
+	return nil
+}
+
+// Text renders the text output
+func (Provider) Text(_ bool, buffer io.Writer) error {
+	return status.RenderText(templatesFS, "clusterchecks.tmpl", buffer, getStatusInfo())
+}
+
+// HTML renders the html output
+func (Provider) HTML(_ bool, _ io.Writer) error {
+	return nil
+}
+
+func populateStatus(stats map[string]interface{}) {
+	if config.Datadog.GetBool("cluster_checks.enabled") {
+		cchecks, err := GetStats()
+
+		if err == nil {
+			stats["clusterchecks"] = cchecks
+		}
+	}
+}
+
+func getStatusInfo() map[string]interface{} {
```

Comment:
Given how simply this function is, I think it might make sense to define everything in the above one.